examples/regional_private_node_pool_oauth_scopes/outputs.tf

```
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "cluster_name" {
+  description = "Cluster name"
+  value       = "${module.gke.name}"
```

Superfluous string interpolation is not necessary with Terraform 0.12. Please apply this to all output values.

```suggestion
  value       = module.gke.name
```